lib/msal-browser/src/broker/nativeBroker/NativeMessageHandler.ts

```diff
@@ -91,11 +103,15 @@ export class NativeMessageHandler {
             channel: NativeConstants.CHANNEL_ID,
             extensionId: this.extensionId,
             responseId: this.responseId++,
-
             body: {
                 method: NativeExtensionMethod.HandshakeRequest
             }
         };
+        this.handshakeEvent.addStaticFields({
+            extensionChannel: NativeConstants.CHANNEL_ID,
```

Comment:
Since this is a static constant do we really need to log this?

Reply:
Good point, thanks. Removed it.